lewis/adapters/stream.py

```diff
@@ -86,7 +84,44 @@ def handle_accept(self):
             StreamHandler(sock, self.target)
 
 
-class Cmd(object):
+class CanNotProcessException(Exception):
+    pass
+
+
+class BaseCommand(object):
+    member = None
+    argument_mappings = None
+    return_mapping = None
```

Comment:
True, did not consider video filters and so on (that was probably the chemist speaking). I kind of like the term `transform`, because that's what it actually is right?

`argument_transforms`, `return_transform`?